src/Sylius/Bundle/ShopBundle/Twig/OrderItemsSubtotalExtension.php

```diff
@@ -13,11 +13,19 @@
 
 namespace Sylius\Bundle\ShopBundle\Twig;
 
+use Sylius\Bundle\ShopBundle\Calculator\OrderItemsSubtotalCalculatorInterface;
 use Sylius\Component\Core\Model\OrderInterface;
-use Sylius\Component\Core\Model\OrderItemInterface;
 
 class OrderItemsSubtotalExtension extends \Twig_Extension
 {
+    /** @var OrderItemsSubtotalCalculatorInterface */
+    private $calculator;
+
+    public function __construct(OrderItemsSubtotalCalculatorInterface $calculator)
```

Comment:
It's BC break :( We should keep current logic as a fallback

Reply:
@lchrusciel Thanks for the review! I addressed the BC break, please have another look.